spk/synocli-disk/Makefile

```
@@ -1,6 +1,6 @@
 SPK_NAME = synocli-disk
-SPK_VERS = 3.3
-SPK_REV = 9
+SPK_VERS = 3.4
```

IMHO we do not need to update the Version if nothing has changed. SPK_REV increment is enough.

```suggestion
SPK_VERS = 3.3
```